src/content/docs/dropins/cart/tutorials/add-product-lines-to-cart-summary.mdx

```diff
@@ -0,0 +1,140 @@
+
+## Step-by-step
+
+The following steps describe how to modify the [commerce-cart.js](https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/blocks/commerce-cart/commerce-cart.js) block file in the boilerplate template to add custom content to the `CartSummaryList` container.
+
+<Tasks>
+
+<Task>
+
+### Add text from a custom product attribute
+
+In this task, we'll add text that provides shipping information when certain conditions apply. For example, an item might be out of stock, and therefore cannot be shipped immediately. Or maybe the product is on clearance and cannot be returned. The `CartSummaryList` component is extended to display text defined by a merchant in the Admin using a custom product attribute. If the custom product attribute is not assigned to a product, then no additional information is displayed.
+
+The following images show how these custom lines can be rendered:
+
+<Diagram caption="">
+   ![](@images/dropins/cart/cart-product-line-shipping.png)
+</Diagram>
+
+<Diagram caption="">
+   ![](@images/dropins/cart/cart-product-line-final.png)
+</Diagram>
+
+<Steps>
+
+1. Open the `blocks/commerce-cart/commerce-cart.js` boilerplate file. This file imports the `CartSummaryList` container, and we want to use a slot to display the custom product attribute. Find the `provider.render(CartSummaryList, {` line in the file and insert a `ProductAttributes` slot with the following code:
+
+   ```javascript
+    slots: {
+      ProductAttributes: (ctx) => {
+        // Prepend Product Attributes
+        const ProductAttributes = ctx.item?.productAttributes;
+
+        ProductAttributes?.forEach((attr) => {
+          if(attr.code === "Fashion Material" || attr.code === "Fashion Style") {
+            if(attr.selected_options) {
+              const selectedOptions = attr.selected_options
+              .filter((option) => option.label.trim() !== '')
+              .map((option) => option.label)
+              .join(', ');
+              
+              if(selectedOptions) {
+                const productAttribute = document.createElement('div');
+                productAttribute.innerText = `${attr.code}: ${selectedOptions}`;
+                ctx.appendChild(productAttribute);
+              }
+            } else if (attr.value) {
+              const productAttribute = document.createElement('div');
+              productAttribute.innerText = `${attr.code}: ${attr.value}`;
+              ctx.appendChild(productAttribute);
+            }
+          }
+        })
+      },
+   ```
+
+   This code creates a slot named `ProductAttributes` that displays the custom product attribute `Shipping Notes`, if it is assigned to a product. If the corresponding attribute is found, the slot creates a new `div` element and appends the attribute code and value to the element. The element is then appended to the `ctx` element, which is the product line in the cart summary.
```

Comment:
Looks like this code block is looking for Fashion Material OR Fashion Style attributes, instead of Shipping Notes